development-utility/development-server/src/com/thoughtworks/go/server/DevelopmentServer.java

```diff
@@ -39,8 +40,9 @@
 
 public class DevelopmentServer {
     public static void main(String[] args) throws Exception {
+        LogConfigurator logConfigurator = new LogConfigurator(DEFAULT_LOG4J_CONFIGURATION_FILE);
+        logConfigurator.initialize();
```

Comment:
No need for this. `LoggingInitializer` will take care of it. The statement `Could not find file 'config/log4j.properties'. Attempting to load from classpath.` appears twice otherwise